File: python/inheritance/classAnimal.py
class Animal:
  def __init__(self, name, age):
    self.name = name
    self.age = age
    self.isAlive = True
  
  def eat(self):
    print("The animal is eating")

  def animalInfo(self):
    print(f"Name: {self.name}")
    print(f"Age: {self.age}")
    print(f"Is this animal alive: {self.isAlive}")

File: javascript/inheritance/classAnimal.js
class Animal {
  constructor(name, age) {
    this.name = name;
    this.age = age;
    this.isAlive = true;
  }

  eat() {
    console.log("The animal is eating");
  }

  animalInfo() {
    console.log(`Name: ${this.name}`);
    console.log(`Age: ${this.age}`);
    console.log(`Is this animal alive ${this.isAlive}`);
  }
}

module.exports = Animal;